apps/api/src/order/order.service.ts

```diff
@@ -20,20 +20,28 @@ export class OrderService {
     @InjectModel(Order.name) private readonly orderModel: Model<Order>,
     private cartService: CartService,
     private productService: ProductsService,
-    private jwtSerivce: JwtService,
+    private jwtService: JwtService,
     private userService: UsersService,
     private authService: AuthService,
   ) {}
 
-  // Objectives:
-  // Merchant Shipping Fee
-  // Product Matching - for loop userCart
-  // Product Inventory - Update Quantity
-  // Computation for Total Price
-  // Delete Items from Cart
+  async findOne(id: any): Promise<Order> {
+    const order = await this.orderModel.findOne({ _id: id });
+
+    if (!order) {
+      throw new NotFoundException('Order is not found');
+    }
+
+    return order;
+  }
+
+  async findAllOrders(req: any): Promise<Order[]> {
+    const userId = await extractIdFromToken(req, this.jwtService);
```

Comment:
req should already have the authId. no need to extract:

use the authId to find the user(same with prev comment)